src/components/Results/Results.js

```diff
@@ -5,8 +5,8 @@ import { Wiki2VecScatter2d, Wiki2VecScatter3d } from "../plots/wiki2vec";
 const Results = () => {
   return (
     <div className={styles.graphContainer}>
-      <Wiki2VecScatter2d queryWord="QueryWord" />
-      <Wiki2VecScatter3d queryWord="QueryWord" />
+      {/* <Wiki2VecScatter2d queryWord="QueryWord" />
```

Comment:
You could remove Wiki2VecScatter2d as we are no longer planning to have it near in future